src/index.js

```diff
@@ -10,3 +12,5 @@ window.SATURN_VERSION = SATURN_VERSION
 marked.setOptions({ sanitize: true, sanitizer: _.escape })
 
 ReactDOM.render(h(Main), document.getElementById('root'))
+
+loadConfig().then(initializeAuth)
```

Comment:
Will this show a spinner while the config and auth load?

Reply:
Yes, because we're still able to mount the root immediately, which shows a spinner by default.